src/editors/containers/ProblemEditor/data/OLXParser.js

```diff
@@ -63,25 +63,27 @@ export const stripNonTextTags = ({ input, tag }) => {
 export class OLXParser {
   constructor(olxString) {
     this.problem = {};
-    this.questionData = {};
-    const questionOptions = {
+    this.problemArray = [];
+    const parserOptions = {
```

Comment:
there should be a comment here explaining the three different options and why they are different